### PR TITLE
Rot 297 ux/UI nutrition details see next 

### PR DESCRIPTION
**Pull request by callumdixon-and**
See next screen

Nutrition endpoints are now prefixed by /Nutrition

If a user naviages to this /Nutrition page on its own, then ir will show the Nutrition screen without the food options

### CODE REVIEW

File: src/api/FirebaseAPI.ts [hunk 1]
```diff
@@ -29,30 +36,13 @@ class FirebaseAPI {
...
+        Object.entries(FoodDetailsComponentData).map(async (each) => {
+          await setDoc(doc(fStore, "FYPData", each[0]), each[1]);
+        })
     );
   };
 
-  static fetchNutritionData = async (name: string):Promise<NutritionalDetailsFirebaseProps> => {
-    const querySnapshot = await getDocs(collection(fStore, "NutritionData", name , "Content"));
-
-    // @ts-ignore
```
Comment (hannah-whelan) [inline src/api/FirebaseAPI.ts:42]: Is it possible to avoid the ts-ignore lines, as we have been trying to remove linting errors where possible

Comment (callumdixon-and) [reply src/api/FirebaseAPI.ts:42]: yes

Comment (hannah-whelan) [reply src/api/FirebaseAPI.ts:42]: Otherwise looks good to me!

Comment (tom-downes-and) [reply src/api/FirebaseAPI.ts:42]: Yeah, no ignores unless necessary! If they are really needed then please add another comment explaining why.

Comment (callumdixon-and) [reply src/api/FirebaseAPI.ts:42]: i have no ideaw why they were there cos there is no linting errors anyway :)

Comment (callumdixon-and) [reply src/api/FirebaseAPI.ts:42]: @hannah-whelan @tom-downes-and please approve this PR when free to do so

**Review by hannah-whelan — APPROVED** [review]

**Pull request merged**